jobsdb/jobsdb.go

```diff
@@ -3315,70 +3315,70 @@ func (jd *HandleT) getBackUpQuery(backupDSRange *dataSetRangeT, isJobStatusTable
 			 		'error_response', dump_table.error_response,
 			 		'parameters', dump_table.parameters
 	)
-			FROM 
+			FROM
 				(
-				SELECT 
+				SELECT
 					*
-				FROM 
-					"%[1]s" 
-				ORDER BY 
-					job_id ASC 
-				LIMIT 
-					%[2]d 
-				OFFSET 
+				FROM
+					"%[1]s"
```

Comment:
Please refrain from doing `"%s"` because DeepSource will complain about it. You can use `%q` without quotes instead.

Reply:
https://github.com/rudderlabs/rudder-server/pull/2252